libs/api/src/lib/graphql/mutation.public.ts

```diff
@@ -581,8 +589,11 @@ export const GraphQLPublicMutation = new GraphQLObjectType<undefined, Context>({
         const subscription = (await prisma.subscription.findUnique({
           where: {
             id: subscriptionId
+          },
+          include: {
+            memberPlan: true
           }
-        })) as SubscriptionWithRelations
+        })) as SubscriptionWithRelations & {memberPlan: MemberPlan}
```

Comment:
Why do we do this explicit casting should prisma not return the type save type?

Reply:
Typescript forced me to add this line.